src/app/parcels/ParcelsPage.tsx

```diff
@@ -116,12 +116,13 @@ const fetchAndFormatParcelTablesData = async (
 };
 
 const ParcelsPage: React.FC<{}> = () => {
-    const today = dayjs();
+    const todayStart = dayjs().startOf("day");
+    const todayEnd = dayjs().endOf("day");
 
     const [isLoading, setIsLoading] = useState(true);
     const [packingDateRange, setPackingDateRange] = useState<DateRangeState>({
-        from: today,
-        to: today,
+        from: todayStart,
+        to: todayEnd,
```

Comment:
Thanks for these changes! 

What happens when you change the "from" / "to" dates on the page? Would the dates be set to 
- the start of the "from" date and 
- the end of the "to" date
correctly?

Reply:
seems like this is already handled :)